packages/terra-status-view/src/StatusView.jsx

```diff
@@ -118,9 +118,9 @@ const StatusView = ({
   let messageSection;
   if (message) {
     messageSection = (
-      <div className={cx('message')}>
+      <p className={cx('message')}>
```

Comment:
Regarding the screenshot changes, we should be able to style the paragraph tag to realign with the styles the div was providing previously.

Reply:
Styles adjusted [here](544daee1958f769bd61b6c9d1d10882921de08ea).